src/pages/Trade/Trade.tsx

```diff
@@ -20,7 +20,7 @@ export const Trade = memo(() => {
         justifyContent='center'
         gap={4}
       >
-        <MultiHopTrade />
+        <MultiHopTrade isRewritingUrl={true} />
```

Comment:
nitpick: can make the diff 6 chars less if you feel like it

Reply:
Nah man, nice joke but it was 7 chars...

Reply:
Removed in [04673b4](https://github.com/shapeshift/web/pull/8708/commits/04673b4e433c4d17da3ee076b00c2ca0d352a2d1)